### PR TITLE
Jm 162792858 set envt variable sendprodinvoice

### PR DESCRIPTION
**Pull request by jacquelineIO**
## Description

This change sets the default for `env` `SEND_PROD_INVOICE`. The file `prod.env` is the only file where `SEND_PROD_INVOICE=true`. When `SEND_PROD_INVOICE=true`, a `P` usage indicator will be used in the generated Invoice (858C EDI). When `SEND_PROD_INVOICE=false`, a `T` usage indicator will be used.

## Reviewer Notes

none.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
go test ./pkg/edi/invoice -testify.m TestGenerate858C
```

## Code Review Verification Steps

* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/162792858) for this change

## Screenshots

none.

### CODE REVIEW

File: cmd/webserver/main_test.go [hunk 1]
```diff
@@ -105,6 +105,10 @@ func (suite *webServerSuite) TestConfigGEX() {
 	suite.Nil(checkGEX(suite.viper))
 }
 
+func (suite *webServerSuite) TestSendProductionEDI() {
+	suite.Nil(checkSendProductionEDI(suite.viper))
+}
```
[the remark this answers is not in the document]
Comment (chrisgilmerproj) [reply cmd/webserver/main_test.go]: This test suite is mostly for checking that the variables you pass in don't throw errors.  However, we could add more tests if there's code in `main.go` that doesn't exist elsewhere.

Also, I'd rename this to `TestConfigSendProductionEDI` like the others.

File: config/env/experimental.env [hunk 2]
```diff
@@ -5,4 +5,5 @@ IWS_RBS_HOST=pkict.dmdc.osd.mil
 HTTP_SDDC_SERVER_NAME=mymove-experimental.sddc.army.mil
 DPS_REDIRECT_URL=https://dpstest.sddc.army.mil/cust
 DPS_COOKIE_NAME=DPSIVV
-GEX_URL=https://gexweba.daas.dla.mil/msg_data/submit/
\ No newline at end of file
+GEX_URL=https://gexweba.daas.dla.mil/msg_data/submit/
+SEND_PROD_INVOICE=false
```
Comment (chrisgilmerproj) [inline config/env/experimental.env]: If you deploy to experimental env for testing would you need this set to `true`?

File: pkg/edi/invoice/generator_test.go [hunk 3]
```diff
@@ -6,10 +6,14 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/go-openapi/swag"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
```
Comment (chrisgilmerproj) [inline pkg/edi/invoice/generator_test.go]: `pflag` and `viper` are replacements for `flag` as imported above.  If you use them then I'd replace line https://github.com/transcom/mymove/pull/1615/files#diff-855527e52724b2167268e651bc3ff58fR36 as well so you're using only one type of CLI flag library.

File: config/env/experimental.env (hunk 2 above)
Comment (jacquelineIO) [reply config/env/experimental.env]: @chrisgilmerproj No, I would leave this as false. `SEND_PROD_INVOICE` simply toggle a flag within the generated invoice from `P>T` for Production vs Test Invoice. If the invoice is tagged at Test, GEX will handle as a fake invoice.

Comment (jacquelineIO) [reply config/env/experimental.env]: @kimallen I have added your GEX_URL to experimental env, I believe this is the correct thing to do @akostibas

**Comment by jacquelineIO** [comment]
@chrisgilmerproj removing `flag` for `pflag`

File: config/env/experimental.env (hunk 2 above)
Comment (kimallen) [reply config/env/experimental.env]: Was it not already there? I see it in the master branch.  Regardless, yes, it should be there.

Comment (jacquelineIO) [reply config/env/experimental.env]: @kimallen I will double-check maybe it was. Thank you for checking!

Comment (jacquelineIO) [reply config/env/experimental.env]: @kimallen I don't see, I just made it the same as prod and staging env

File: cmd/webserver/main_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jacquelineIO) [reply cmd/webserver/main_test.go]: I have removed TestConfigSendProductionEDI @chrisgilmerproj  based on Patrick's comment below.

**Pull request merged**